docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+# SPDX-FileCopyrightText: 2025 INFO.nl
+# SPDX-License-Identifier: EUPL-1.2+
+version: "3.8"
+services:
+  frontend:
+    build:
+      context: ./office-add-in
+      dockerfile: Dockerfile
+    image: my-office-addin-frontend
+    container_name: my-office-frontend
+    ports:
+      - "3000:3000"
+    networks:
+      - office-add-in-net
+
+  backend:
+    build:
```

Comment:
I'm not sure we should use a build step in the compose setup in the future. For now, let's do it this way, then we can start doing some e2e testing...